<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hanile_log_main</class>
 <widget class="QWidget" name="hanile_log_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>795</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>기준값 설정값</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="MCB_standard">
       <item>
        <property name="text">
         <string>Rate</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>설정값</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_standard_value">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>시간 설정(S)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_standard_time">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="main_log_layout"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Vacuum</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Thickness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="MCB_rate">
       <property name="text">
        <string>Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="MCB_power">
       <property name="text">
        <string>Power</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="text">
        <string>Xtal No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="text">
        <string>Xtal Life</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>Xtal Freq</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="text">
        <string>XPos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="text">
        <string>YPos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="item_table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <column>
      <property name="text">
       <string>List</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Lot_id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>mat_name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>테스트체크</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="MZ_reset_btn">
       <property name="text">
        <string>줌리셋</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
